src/arrays/multidimensional_arrays/hard/q2/SimpleDisplay.java

```diff
@@ -0,0 +1,50 @@
+package arrays.multidimensional_arrays.hard.q2;
+
+/*
+Write a Java program called SimpleDisplay. It should have a main method which asks the user for the number of rows,
+columns, and array values, and passes that array to a method called simpleDisplay. The method simpleDisplay should
+accept a multidimensional 2D array (int [][] nums), and count the number of elements in it that are even. The count is
+printed out in the method. This should work for any size 2D array.
+
+Sample output:
+    Enter the number of rows:
+    3
+    Enter the number of columns:
+    2
+    Enter the values for the array:
+    1 2 3 4 5 6
+    count of even numbers in array = 3
```

Comment:
The output here should make grammatical sense, maybe more like "There are x even values in the array."